cranelift-codegen/Cargo.toml

```diff
@@ -53,12 +53,20 @@ core = [
 # can significantly increase the size of the library.
 testing_hooks = []
 
-# ISA targets for which we should build.
+# ISA targets for which we should build. At least one must be enabled.
```

Comment:
Give build.rs as it is now, we should change this comment to "if no ISA targets are explicitly enabled, the ISA target for the host machine is enabled".